srcpkgs/meson/patches/dont-use-native-pkgocnfig-for-gir.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/mesonbuild/modules/gnome.py b/mesonbuild/modules/gnome.py
```

Comment:
Typo in filename:
dont-use-native-pkg`oc`nfig-for-gir.patch -> dont-use-native-pkg`co`nfig-for-gir.patch